AdyenActions/UI/View Controllers/Document/DocumentActionView.swift

```diff
@@ -75,11 +74,20 @@ internal final class DocumentActionView: UIView {
     /// The UI style.
     private let style: DocumentComponentStyle
     
-    internal init(viewModel: DocumentActionViewModel, style: DocumentComponentStyle) {
+    internal init(
+        viewModel: DocumentActionViewModel,
+        style: DocumentComponentStyle,
+        imageLoader: ImageLoading = ImageLoaderProvider.imageLoader()
+    ) {
         self.viewModel = viewModel
         self.style = style
         super.init(frame: .zero)
         configureViews()
+        
+        imageView.load(
```

Comment:
Ideally we want to trigger load on view appear and not instantiation. However, I'm not familiar with this part of the codebase so please make your own judgement. I won't comment on other similar cases.

Reply:
Yeah we could override `didMoveToWindow()` and see whether it became visible/invisible. What do you think?

Reply:
I am not sure as this method is part of `UIView` lifecycle. I'd love to have full control over the lifecycle and the best we can go with is UIViewController or table/collection view